clinica/iotools/converters/genfi_to_bids/genfi_to_bids.py

```diff
@@ -9,6 +9,7 @@ def convert_images(
     bids_dir: PathLike,
     path_to_clinical: Optional[PathLike],
     gif: bool,
+    path_to_clinical_tsv: Optional[PathLike] = None,
```

Comment:
Could you also add the `= None` to the `path_to_clinical` argument as well ?

Also, is `gif` optional ? If not, it should be before the optional arguments, if yes, it should be declared as optional with a default value.